src/Miunie.Core/Providers/MiunieUserProvider.cs

```diff
@@ -31,7 +33,12 @@ public void StoreUser(MiunieUser user)
             }
         }
 
-        private MiunieUser EnsureExistence(
+        public IEnumerable<MiunieUser> GetAllUsers()
+        {
+            return _persistentStorage.RestoreAll<MiunieUser>();
+        }
+
+        private MiunieUser Exists(
```

Comment:
Not sure if an accidental change but makes more sense to have it as EnsureExistence or at-least EnsureExists